src/Backend/WASMWrapper.js

```diff
@@ -1,6 +1,18 @@
 import DataSeriesFactory from './DataSeriesFactory.js'
 import Module from './weblibdedx.js'
 
+
+/**
+ * @typedef {LibdedxEntity}
+ * @property {number} code - the libdedx ID assigned to the entity
```

Comment:
can't we name rename `code` to `id` or `code_number` ? Code sounds like sourcecode.